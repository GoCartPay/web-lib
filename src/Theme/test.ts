import theme from './index'

// Note that if the design token changes a used value, the snapshot will need to be updated
it('Produces correct output', () => {
  expect(theme).toMatchSnapshot()
})
