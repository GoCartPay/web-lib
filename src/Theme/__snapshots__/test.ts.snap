// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Produces correct output 1`] = `
Object {
  "breakpoints": Object {
    "between": [Function],
    "down": [Function],
    "keys": Array [
      "xs",
      "sm",
      "md",
      "lg",
      "xl",
    ],
    "not": [Function],
    "only": [Function],
    "unit": "px",
    "up": [Function],
    "values": Object {
      "lg": 1200,
      "md": 900,
      "sm": 600,
      "xl": 1536,
      "xs": 0,
    },
  },
  "components": Object {
    "MuiCssBaseline": Object {
      "styleOverrides": "
  @font-face {
    font-family: 'TWK Lausanne';
    font-style: normal;
    font-display: swap;
    font-weight: 400;
    src: url() format('truetype');
    unicodeRange: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF;
  }
",
    },
  },
  "direction": "ltr",
  "mixins": Object {
    "toolbar": Object {
      "@media (min-width:0px) and (orientation: landscape)": Object {
        "minHeight": 48,
      },
      "@media (min-width:600px)": Object {
        "minHeight": 64,
      },
      "minHeight": 56,
    },
  },
  "palette": Object {
    "action": Object {
      "activatedOpacity": 0.12,
      "active": "#757575ff",
      "disabled": "#bdbdbdff",
      "disabledBackground": "rgba(0, 0, 0, 0.12)",
      "disabledOpacity": 0.38,
      "disabledbackground": "#e0e0e0ff",
      "focus": "rgba(0, 0, 0, 0.12)",
      "focusOpacity": 0.12,
      "focus_12p": "#419590ff",
      "hover": "#000000ff",
      "hoverOpacity": 0.04,
      "selected": "#ebebebff",
      "selectedOpacity": 0.08,
      "shades": undefined,
    },
    "augmentColor": [Function],
    "background": Object {
      "default": "#fafafaff",
      "paper": "#ffffffff",
    },
    "common": Object {
      "black": undefined,
      "white": undefined,
    },
    "contrastThreshold": 3,
    "divider": "rgba(0, 0, 0, 0.12)",
    "error": Object {
      "contrast": "#ffffffff",
      "contrastText": "#fff",
      "dark": "#c62828ff",
      "light": "#ef5350ff",
      "main": "#d32f2fff",
    },
    "getContrastText": [Function],
    "grey": Object {
      "100": "#f5f5f5",
      "200": "#eeeeee",
      "300": "#e0e0e0",
      "400": "#bdbdbd",
      "50": "#fafafa",
      "500": "#9e9e9e",
      "600": "#757575",
      "700": "#616161",
      "800": "#424242",
      "900": "#212121",
      "A100": "#f5f5f5",
      "A200": "#eeeeee",
      "A400": "#bdbdbd",
      "A700": "#616161",
    },
    "info": Object {
      "contrast": "#ffffffff",
      "contrastText": "#fff",
      "dark": "#01579bff",
      "light": "#03a9f4ff",
      "main": "#0288d1ff",
    },
    "mode": "light",
    "other": Object {
      "backdrop_overlay": "#808080ff",
      "divider": "#e7e7e8ff",
      "filled_input_background": "#f0f0f0ff",
      "outlined_border-23p": "#c4c4c4ff",
      "rating active": "#ffb400ff",
      "snackbar": "#323232ff",
      "standard_input_line": "#949494ff",
    },
    "primary": Object {
      "contrastText": "#fff",
      "contrast_text": "#ffffffff",
      "dark": "#000000ff",
      "light": "#717174ff",
      "main": "#121317ff",
    },
    "secondary": Object {
      "contrast": "#ffffffff",
      "contrastText": "#fff",
      "dark": "#044d47ff",
      "light": "#419590ff",
      "main": "#117b74ff",
    },
    "success": Object {
      "contrast": "#ffffffff",
      "contrastText": "#fff",
      "dark": "#1b5e20ff",
      "light": "#4caf50ff",
      "main": "#2e7d32ff",
    },
    "text": Object {
      "disabled": "#a5a5a7ff",
      "primary": "#121317ff",
      "secondary": "#717174ff",
    },
    "tonalOffset": 0.2,
    "warning": Object {
      "contrast": "#ffffffff",
      "contrastText": "#fff",
      "dark": "#e65100ff",
      "light": "#ff9800ff",
      "main": "#ed6c02ff",
    },
  },
  "shadows": Array [
    "none",
    "0px 1px 2px 0px #1018280d",
    "0px 1px 3px 0px #1018281a",
    "0px 2px 4px -2px #1018280f, 0px 4px 8px -2px #1018281a",
    "0px 4px 6px -2px #10182808, 0px 12px 16px -4px #10182814",
    "0px 8px 8px -4px #10182808, 0px 20px 24px -4px #10182814",
    "0px 24px 48px -12px #1018282e",
    "0px 32px 64px -12px #10182824",
    "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
    "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
    "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
    "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
    "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
    "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
    "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
    "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
    "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
    "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
    "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
    "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
    "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
    "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
    "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
    "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
    "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
  ],
  "shape": Object {
    "borderRadius": "20px",
  },
  "spacing": [Function],
  "transitions": Object {
    "create": [Function],
    "duration": Object {
      "complex": 375,
      "enteringScreen": 225,
      "leavingScreen": 195,
      "short": 250,
      "shorter": 200,
      "shortest": 150,
      "standard": 300,
    },
    "easing": Object {
      "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
      "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
      "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
      "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
    },
    "getAutoHeightDuration": [Function],
  },
  "typography": Object {
    "allVariants": undefined,
    "body1": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 16,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "lineHeight": 1.5,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "body2": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 14,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "lineHeight": 1.43,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "button": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": "0.875rem",
      "fontWeight": 500,
      "lineHeight": 1.75,
      "textTransform": "uppercase",
    },
    "caption": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 12,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0.4,
      "lineHeight": 1.66,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "fontFamily": "TWK Lausanne",
    "fontSize": 14,
    "fontWeightBold": 700,
    "fontWeightLight": 300,
    "fontWeightMedium": 500,
    "fontWeightRegular": 400,
    "h1": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 64,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": -2,
      "lineHeight": 1.167,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "h2": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 48,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": -2,
      "lineHeight": 1.2,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "h3": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 36,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": -1.8,
      "lineHeight": 1.167,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "h4": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 30,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": -1.8,
      "lineHeight": 1.235,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "h5": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 24,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "lineHeight": 1.334,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "h6": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 20,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "lineHeight": 1.6,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "htmlFontSize": 16,
    "legal": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 9,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "legal_url": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 9,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "underline",
    },
    "overline": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 12,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 1.2,
      "lineHeight": 2.66,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "uppercase",
      "textDecoration": "none",
      "textTransform": "uppercase",
    },
    "pxToRem": [Function],
    "subtitle1": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 16,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "lineHeight": 1.75,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "subtitle2": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 14,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": 0,
      "lineHeight": 1.57,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
    "title": Object {
      "fontFamily": "TWK Lausanne",
      "fontSize": 96,
      "fontStretch": "normal",
      "fontStyle": "normal",
      "fontWeight": 400,
      "letterSpacing": -2,
      "paragraphIndent": 0,
      "paragraphSpacing": 0,
      "textCase": "none",
      "textDecoration": "none",
    },
  },
  "zIndex": Object {
    "appBar": 1100,
    "drawer": 1200,
    "fab": 1050,
    "mobileStepper": 1000,
    "modal": 1300,
    "snackbar": 1400,
    "speedDial": 1050,
    "tooltip": 1500,
  },
}
`;
